cost/google/cud_report/README.md

```diff
@@ -0,0 +1,34 @@
+## Google Committed Use Discount (CUD) Report
+ 
```

Comment:
Need to add a note similar to the other RightScale account independent policies.
https://github.com/rightscale/policy_templates/tree/master/cost/budget_alerts

**As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account.**

Reply:
@ryanoleary Should we specify "single account" as opposed to "master account"?